src/js/player.js

```diff
@@ -4290,6 +4292,32 @@ class Player extends Component {
     return !!this.isAudio_;
   }
 
+  /**
+   * Get the current audioPosterMode state or set audioPosterMode to true or false
+   *
+   * @param {boolean} [value]
+   *         The value to set audioPosterMode to.
+   *
+   * @return {boolean}
+   *         True if audioPosterMode is on, false otherwise.
+   */
+  audioPosterMode(value) {
+
+    if (value === undefined) {
+      return this.options_.audioPosterMode;
+    }
+    this.options_.audioPosterMode = !!value && !this.options_.audioOnlyMode;
```

Comment:
I'm having trouble following the logic here. Do you think this would be clearer?
```suggestion
    value = Boolean(value);

    // If no value is provided or it is unchanged, acts as a getter.
    if (value === undefined || value === this.options_.audioOnlyMode) {
      return this.options_.audioOnlyMode;
    }
```